### ISSUE TITLE
Public, Qt-independent API for dimension playing

### ISSUE THREAD

**Issue by faymanns** [napari/napari#6274]
## 🚀 Feature
The API for starting and stopping the animation on the dimension sliders should be made public before `qt_viewer` is removed.

## Motivation
Currently the API for playing one of the dimensions is `viewer.window.qt_viewer.dims.play()`. Once `qt_viewer` is removed this will no longer be possible and the dimension sliders can no longer be played from within a plugin.

## Pitch
 It is probably not possible to have an event-loop-independent way to animate the dimension playback but the playback API could be exposed as e.g. `viewer.window.play_dim(axis=0, fps=10)` and call the window backend from there.

## Additional context
This issue was first discussed on [image.sc](https://forum.image.sc/t/starting-napari-playback-from-within-a-plugin/86687).

**Comment by melonora** [comment]
Hi @faymanns,

Thank you for opening the issue! One question, is this something you would like to work on yourself as a contribution to napari? If so we can help you get started:)

**Comment by faymanns** [comment]
Hi @melonora,

Yes, I would like work it myself if possible. Is there already a branch or pull request for removing public access to `qt_viewer` or should I fork the main branch?

**Comment by Czaki** [comment]
No, there is no such PR, as still some API are not implemented. 
It is best to have a release that contains both missed API and `qt_viewer` for a transition period. 

So jus use main branch.

**Comment by melonora** [comment]
Hi Faymanns, Thank you for being willing to contribute! If you want to discuss some time, feel free to join one of the community meetings: https://napari.org/stable/community/meeting_schedule.html. This week there might not be one or at least at a different time as most of the team is in San Francisco at the moment for a hackathon. In any case if you have any questions feel free to get in touch. You can also reach the developers [here](napari.zulipchat.com).

**Comment by faymanns** [comment]
Thanks! I will try to join the community meeting next week. In the meantime I have taken a shot at it and submitted a pull request (#6300).
